src/styles/variables.scss

```diff
@@ -45,6 +46,7 @@ $color-success-dark: #00d3a1;
 $color-success-hover: #00dc87;
 $color-warning: #ffb422;
 $color-danger: #ff2b5e;
+$color-special: #ff4746;
```

Comment:
I talked with Tsvetan about this and the outcome of the conversation is as follows:

1. The color you used here should be used as the `$color-danger`. So from now on all "danger" places will use it.
2. The color that we actually use as the "danger" (`#ff2b5e`) should be moved to the new variable `$color-secondary` and should be used for the purpose you created `$color-special`.
3. `$color-special` should be removed.
4. The `BtnMain` variant `secondary` should use instead of gray the color `$color-secondary` (the pink).
5. The old `secondary` variant of `BtnMain` should be replaced with `muted`. Please remember to align every occurence of the BtnMain with `variant="secondary"` prop.

Reply:
Resolved in #1741